pipelines/acs_manual_update.py

```diff
@@ -0,0 +1,162 @@
+import argparse
```

Comment:
A general description of the "why" might be nice here. "We recieve data directly from DCP Population in an excel file that necessitates cleaning the data to a standard form OSE uses to ingest the data into Population FactFinder. This script has functions that handle standardizes column schema, etc..."

Reply:
see 99fc971